VideoApp/Video-InternalTests/Specs/PasscodeComponentsSpec.swift

```diff
@@ -100,11 +100,11 @@ class PasscodeComponentsSpec: QuickSpec {
                 }
 
             }
-            
+
             context("when string is invalid") {
                 context("when string length is 6") {
                     it("throws passcodeIncorrect error") {
-                        expect { try PasscodeComponents(string: "256984") }.to(throwError(AuthError.passcodeIncorrect))
+                        expect(expression: { try PasscodeComponents(string: "256984") }).to(throwError(AuthError.passcodeIncorrect))
```

Comment:
This is a workaround for what seems to be a Swift bug. 

Here is a Nimble issue that is tracking it: https://github.com/Quick/Nimble/issues/809

Lets keep an eye on this issue. Hopefully it is fixed in a new beta and we can revert this change.